src/platform/silabs/rs911x/ble_config.h

```diff
@@ -0,0 +1,272 @@
+/*******************************************************************************
+ * @file  rsi_ble_config.h

```

Comment:
Copy paste from rsi_ble_config.h please remove this files completely....

Reply:
Removed this file from this PR , adding it in a different PR

Reply:
Removed this file from PR , will add in a separate PR